src/Diagnostics.ModelsAndUtils/Models/GeoMaster/GeoMasterResponse.cs

```diff
@@ -14,4 +14,9 @@ public class GeoMasterResponse
         public IDictionary<string, dynamic> Properties { get; set; }
     }
 
+    public class GeoMasterResponseValue
```

Comment:
same here. Is this class getting used by scripts or just the data provider

Reply:
Once we add a pass through kind of method, the detectors may benefit directly with this class so I am thinking of leaving this one as is